index.js

```diff
@@ -0,0 +1,11 @@
+(() => {
+  const mainElement = document.getElementsByTagName("main");
+  console.log(mainElement[0]);
+
+  const h1Element = document.createElement("h1");
+  h1Element.innerText = "Hello World!";
+
+  setTimeout(() => {
```

Comment:
```suggestion
  // １秒後に要素を挿入する処理
  setTimeout(() => {
```

commentを挿入するとわかりやすいかもです。